Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -211,16 +212,18 @@ extension ActivityStreamPanel {
                 currentTraits = UITraitCollection(horizontalSizeClass: .compact)
             }
             switch self {
-            case .pocket:
+            case .pocket, .libraryShortcuts:
+                let window = UIApplication.shared.keyWindow
+                let safeAreaInsets = window?.safeAreaInsets.left
                 var insets = ASPanelUX.sectionInsetsForSizeClass[currentTraits.horizontalSizeClass]
-                insets = insets + ASPanelUX.MinimumInsets
+                insets = insets + ASPanelUX.MinimumInsets + (safeAreaInsets ?? 0)
                 return insets
             case .topSites:
-                return ASPanelUX.sectionInsetsForSizeClass[currentTraits.horizontalSizeClass]
-            case .libraryShortcuts:
-                let shortcutsWidth = min(ASPanelUX.LibraryShortcutsMaxWidth, frameWidth)
-                return (frameWidth - shortcutsWidth)/2
+                var insets = ASPanelUX.sectionInsetsForSizeClass[currentTraits.horizontalSizeClass]
+                insets = insets + ASPanelUX.TopSitesInsets
+                return insets
```

Comment:
styling nit: Consider having
`let insets = ASPanelUX.sectionInsetsForSizeClass[currentTraits.horizontalSizeClass]` above the switch, then 
```
case .topSites:
  return insets + ASPanelUX.TopSitesInsets
```